contracts/core/test/xchain/ts/merkle.ts

```diff
@@ -45,13 +45,11 @@ export class XMsgMerkleTree {
     return tree[0]
   }
 
-  // Order msgs by destination chain id, then offset
+  // Order msgs by offset. XMsgs within an xblock are ordered by log index, which implicity orders
+  // by offset (ascending).
```

Comment:
Note this is only correct for blocks with msgs for a single stream. It isn't correct for blocks with msgs for multiple streams.